### ISSUE TITLE
Example config file missing some TSM1 config options

### ISSUE THREAD

**Issue by daviesalex** [influxdb/influxdb#4524]
There are some config options in the code, e.g. from ./tsdb/config.go:
```
        // WAL configuration options for tsm1 introduced in 0.9.5
        WALFlushMemorySizeThreshold int `toml:"wal-flush-memory-size-threshold"`
        WALMaxMemorySizeThreshold   int `toml:"wal-max-memory-size-threshold"`
```
That are not defined in etc/config.sample.toml. Are they referred to by some other name, or are they just missing?

I have not done an extensive comparison; there may be others.

**Comment by daviesalex** [comment]
I was going to send a PR to add these, but its also not clear from the example config what options refer to what storage engines. In the example config for example it says "These are the WAL settings for the storage engine >= 0.9.3" for wal-enable-logging, which implies several engines, but in the code WALLoggingEnabled is under a comment " that says it is for bz1 only. 

I think it would be ideal to add the same comments that are in the code to the sample file... and if the comments in the code are wrong adjusting them.

**Comment by otoolep** [comment]
I believe this has been addressed as of 0.9.6.
